test/studies/lcals/LCALSMain.skipif

```diff
@@ -0,0 +1,3 @@
+# This test takes too long if --fast is not used so only run it for
```

Comment:
Executing takes really long with bounds checks enabled.  The long, medium and short lengths are all intended to run about the same amount of time as each other.  The shorter loops have larger numbers of sampling repetitions to make them take about the same time.  I can look into running these without '--fast' some more.